sdk/ml/azure-ai-ml/tests/test_configs/feature_set/invalid_spec4/FeatureSetSpec.yaml

```diff
@@ -0,0 +1,30 @@
+source:
```

Comment:
can you leave a comment what is invalid for this example?

Reply:
Added.